examples/nextjs/.gitignore

```diff
@@ -7,9 +7,6 @@
 /.pnp
 .pnp.js
 
-# testing
-/coverage
-
```

Comment:
nit: Fine to remove this, but it is unrelated to the coverage tooling in the top-level package.